shared/middleware/helpers/source_bucket.rb

```diff
@@ -1,6 +1,7 @@
 require 'cdo/project_source_json'
 
-MAIN_JSON_FILENAME = 'main.json'.freeze
+# Check whether it is defined, since helpers can be double-included during test and cron tasks.
```

Comment:
In this case the helper is being double included while running dashboard server, so you might just say more generally that they "can sometimes be double-included".